robottelo.properties.sample

```diff
@@ -25,6 +25,9 @@ ssh_key=
 # Admin password when accessing API and UI
 # admin_password=changeme
 
+# Version is taken from here OR SAT_VERSION envvar or SSH call
+# version=6.6
```

Comment:
Do we really need new setting?
I understand why you don't want to depend on version obtained from Satellite itself, but isn't environmental variable enough?